web/js/map/granule/util.js

```diff
@@ -107,23 +129,64 @@ export const getGranuleFootprints = (layer) => {
 };
 
 /**
- * Get CMR query dates for building query string and child processes
+ * Get start/end dates for CMR granule query. We need a broader range
+ * for polar granules since only a few granules from each swath are
+ * visible at the poles.
  *
- * @method getCMRQueryDates
- * @static
  * @param {object} selectedDate - date object
  * @returns {object}
     * @param {object} startQueryDate - date object
     * @param {object} endQueryDate - date object
   */
-export const getCMRQueryDates = (selectedDate) => {
+export const getCMRQueryDates = (crs, selectedDate) => {
   const date = new Date(selectedDate);
-  const startQueryDate = util.dateAdd(date, 'hour', -8);
-  const endQueryDate = util.dateAdd(date, 'hour', 4);
+  if (crs === CRS.GEOGRAPHIC || crs === CRS.WEB_MERCATOR) {
+    return {
+      startQueryDate: util.dateAdd(date, 'hour', -12),
+      endQueryDate: util.dateAdd(date, 'hour', 4),
+    };
+  }
+  // Polar projections
+  return {
+    startQueryDate: util.dateAdd(date, 'hour', -48),
+    endQueryDate: util.dateAdd(date, 'hour', 4),
+  };
+};
+
+/**
+ * Get the URL parameters for a CMR request for granule browse
```

Comment:
The params listed in the definition (date & state) are inconsistent with the parameters required for the getParamsForGranuleRequest function (def, date, crs).